ui/src/views/Provisioners/ViewWorkers/index.jsx

```diff
@@ -3,6 +3,7 @@ import React, { Component, Fragment } from 'react';
 import { graphql } from 'react-apollo';
 import dotProp from 'dot-prop-immutable';
 import { parse, stringify } from 'qs';
+import { get, find } from 'lodash';
```

Comment:
We use https://ramdajs.com/ for functional helper methods. Let's swap to using ramda.

Reply:
@helfi92  Please update review :) Thanks .